UIColor+hexValue.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIColor+hexValue.swift
+//  Rocket.Chat.iOS
+//
+//  Created by Kornelakis Michael on 9/2/15.
+//  Copyright © 2015 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+
+extension UIColor {
+    
+    static func colorWithHexValue(redValue: CGFloat, greenValue: CGFloat, blueValue: CGFloat, alpha: CGFloat) -> UIColor {
```

Comment:
What about creating an extension to return Rocket.Chat colors? Something like...

1. `static func rocketMainFontColor() -> UIColor`
2. `static func rocketSecondaryFontColor() -> UIColor`
3. `static func rocketRedColor() -> UIColor`
4. `static func rocketBlueColor() -> UIColor`